compiled_starters/typescript/app/main.ts

```diff
@@ -0,0 +1,33 @@
+import { TextDecoder } from "util";
+
+function decodeBencode(bencodedValue: Uint8Array): string {
```

Comment:
This looks significantly more complex compared to the Javascript one - no comments on what it does, no examples.. @ryan-gang can we try to make it a bit more readable?